oak_dice/src/utils.rs

```diff
@@ -29,3 +33,16 @@ pub fn cbor_encoded_bytes_to_vec(bytes: &[u8]) -> Result<Vec<u8>, String> {
         .map_err(|err| format!("failed to write bytes: {:?}", err))?;
     Ok(result)
 }
+
+/// Like the slice `copy_from_slice` method but does not panic if slices are not
```

Comment:
```suggestion
/// Like [`slice::copy_from_slice`] but does not panic if slices are not
```

Or something like that